crates/cheatnet/src/rpc.rs

```diff
@@ -661,6 +664,69 @@ pub fn library_call_syscall(
     })
 }
 
+pub enum KeccakOutput {
+    Success { ret_data: KeccakResponse },
+    Panic { panic_data: Vec<Felt252> },
+}
+
+pub fn keccak_syscall(
```

Comment:
Is it copied from somewhere? If so can we leave a comment with path to a file and repo like in the fn below?

Reply:
Yup, it is changed in few places to adjust errors but should be marked as `inspired`